### PR TITLE
hello

### PR DESCRIPTION
**Pull request by NguyenHongHieu**
hello world

### CODE REVIEW

**Comment by NguyenHongHieu** [comment]
hello

File: test.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package number_of_date;
 
-public class NumberOfDay {
+public class numberofday {
```
Comment (huynhphvan) [inline test.java:3]: _**Cái này_ sao lại sửa ? bị ngu à** @NguyenHongHieu

Comment (huynhphvan) [inline test.java:3]: ??? #
```suggestion
public class numberOfDay {
```

File: test.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 	//declare properties
 	private int month,year;
```
Comment (huynhphvan) [inline test.java:6]: làm cái gì vaay

**Comment by huynhphvan** [comment]
- [ ]

**Comment by huynhphvan** [comment]
___
*[test.java, line 15 at r1](https://reviewable.io/reviews/huynhphvan/code-review-test/2#-M4FBrQVBwEhX29Q2I-9:-M4FBrQVBwEhX29Q2I-A:b3ms91h) ([raw file](https://github.com/huynhphvan/code-review-test/blob/a8a52ef1f7afe40f1469da524e8b0a3bd2f55b6a/test.java#L15)):*
> ```Java
> 	}
> 	
> 	public void setMonth(int m) {
> ```

Hay lắm

**Comment by huynhphvan** [comment]
___
*[test.java, line 67 at r1](https://reviewable.io/reviews/huynhphvan/code-review-test/2#-M4FCCEABPNJfgt5OgBZ:-M4FCCEABPNJfgt5OgB_:b2nhabx) ([raw file](https://github.com/huynhphvan/code-review-test/blob/a8a52ef1f7afe40f1469da524e8b0a3bd2f55b6a/test.java#L67)):*
> ```Java
> 		n2.setMonth(4);
> 		n2.setYear(2017);
> 		System.out.println("Number Of Day Is: " + n2.numberOfDay());
> ```

ADu

**Pull request closed without merging**